src/routes/docs/advanced/self-hosting/debug/+page.markdoc

```diff
@@ -46,8 +46,6 @@ To change your dev environment, edit your server `_APP_ENV` environment variable
 
 ```yaml
 ...
-    - influxdb
-    - telegraf
```

Comment:
Lol this section needs to be updated:

````
To change your dev environment, edit your server `_APP_ENV` environment variable from 'production' to 'development' in your Appwrite `.env` file located in the `appwrite` directory where you first installed Appwrite.

```yaml
...
_APP_ENV=development
...
```
````

It would be great if we could syntax highlight it as a diff. I don't know if we support that though.

Reply:
This section wasn't updated.